docs/windows-client.md

```diff
@@ -4,36 +4,21 @@
 
 This documentation has the goal of showing how a user can use the official Windows [Tailscale](https://tailscale.com) client with `headscale`.
 
-## Add registry keys
-
-To make the Windows client behave as expected and to run well with `headscale`, two registry keys **must** be set:
+## Installation
 
-- `HKLM:\SOFTWARE\Tailscale IPN\UnattendedMode` must be set to `always` as a `string` type, to allow Tailscale to run properly in the background
-- `HKLM:\SOFTWARE\Tailscale IPN\LoginURL` must be set to `<YOUR HEADSCALE URL>` as a `string` type, to ensure Tailscale contacts the correct control server.
+Download the [Official Windows Client](https://tailscale.com/download/windows) and install it.
 
-You can set these using the Windows Registry Editor:
+When the installation has finished, close the tailscale application in the windows tray. (Richt click > Exit)
 
-![windows-registry](./images/windows-registry.png)
+## Login to Headscale server
 
-Or via the following Powershell commands (right click Powershell icon and select "Run as administrator"):
+Open Command Prompt or Powershell and type in the following:
 
 ```
-New-Item -Path "HKLM:\SOFTWARE\Tailscale IPN"
-New-ItemProperty -Path 'HKLM:\Software\Tailscale IPN' -Name UnattendedMode -PropertyType String -Value always
-New-ItemProperty -Path 'HKLM:\Software\Tailscale IPN' -Name LoginURL -PropertyType String -Value https://YOUR-HEADSCALE-URL
+tailscale up --accept-routes --login-server https://<your-headscale-server>
```

Comment:
The other examples do not contain, `--accept-routes`. It should be removed for consistency reasons.